vars/common-vars.yml

```diff
@@ -31,7 +31,18 @@ socok8s_repos_to_configure:
   openSUSE-Leap-15.0-Update: "{{ obs_mirror }}/update/leap/15.0/oss/"
   openSUSE-Leap-15.0-Cloud: "{{ obs_mirror }}/repositories/Cloud:/Tools/openSUSE_Leap_15.0/"
   Leap15develtools: "{{ obs_mirror }}/repositories/devel:/tools/openSUSE_Leap_15.0/"
+  SLE-15-SP1-JeOS-GM-Full: http://download.suse.de/full/full-sle15-sp1-x86_64/
```

Comment:
`download.suse.de` only works inside our VPN. Is that the expected behavior?

Reply:
yep. For the moment we dont support using SCC for updates so this will have to work until we iterate over that and enable it

Reply:
This repo is not ideal, but if we make it work, we at least know it's possible. And in another PR's we can work on better setup.